go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/openshift/client-go v3.9.0+incompatible
 	github.com/operator-framework/operator-sdk v0.8.1-0.20190517223317-f7f644008098
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
-	github.com/prometheus/client_golang v1.1.0 // indirect
+	github.com/prometheus/client_golang v1.1.0
```

Comment:
Why has this changed?

Reply:
Not sure. Again I believe the `require` section of `go.mod` file are generated by the `go mod` tool. If I manually revert it it will be added back again.

Reply:
Does it work without modifying the go.mod file?
For what it looks like, it seems they are just updates to existing modules versions that have been applied when performing go mod commands.
If they are not needed let's not add them in the PR. We can take a look at go.mod/go.sum status of the module versions in general in another PR/issue

Reply:
I will try, should be possible. It's just my IDE keeps modifying it automatically.